### PR TITLE
Azure enhancement dev

### PR DESCRIPTION
**Pull request by poornima-metron**

### CODE REVIEW

**Comment by suraj-metron** [comment]
Changelog file is not updated. Please update

**Review by suraj-metron — CHANGES_REQUESTED** [review]

File: src/config.ts [hunk 1]
```diff
@@ -36,6 +36,13 @@ export const ingestionConfig: IntegrationIngestionConfigFieldMap = {
     defaultsToDisabled: false,
     cannotBeDisabled: false,
   },
+  [INGESTION_SOURCE_IDS.APPLICATION_SECURITY_GROUP]: {
```
[the remark this answers is not in the document]
Comment (poornima-metron) [reply src/config.ts:39]: added for all services

File: src/index.ts [hunk 2]
```diff
@@ -36,6 +37,10 @@ import { managementGroupSteps } from './steps/resource-manager/management-groups
 import { containerServicesSteps } from './steps/resource-manager/container-services';
 import { frontdoorSteps } from './steps/resource-manager/frontdoor';
 import { ingestionConfig } from './config';
+import { SynapseSteps } from './steps/resource-manager/synapse';
```
[the remark this answers is not in the document]
Comment (poornima-metron) [reply src/index.ts:40]: Kubernetes is already existing service which is container Service

File: src/steps/resource-manager/express-route/converters.ts [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+  });
+}
+
+export function createAzureExpressRouteEntity(
+  instnaceId: string,
+  subscriptionKey,
+): Entity {
+  return createIntegrationEntity({
+    entityData: {
+      source: {},
```
[the remark this answers is not in the document]
Comment (poornima-metron) [reply src/steps/resource-manager/express-route/converters.ts]: added undefined

File: src/steps/resource-manager/synapse/converter.ts [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+}
+
+/**
+ * create Synapse service entity.
+ * @returns  Synapse Entity
+ */
+export function createSynapseServiceEntity(instnaceId: string): Entity {
+  return createIntegrationEntity({
+    entityData: {
+      source: {},
```
[the remark this answers is not in the document]
Comment (poornima-metron) [reply src/steps/resource-manager/synapse/converter.ts]: added undefined

**Review by suraj-metron — APPROVED** [review]

**Pull request merged**